x-pack/plugins/security_solution/cypress/tasks/create_new_rule.ts

```diff
@@ -190,6 +196,27 @@ export const fillDefineCustomRuleWithImportedQueryAndContinue = (
   cy.get(CUSTOM_QUERY_INPUT).should('not.exist');
 };
 
+export const fillScheduleRuleAndContinue = (rule: CustomRule | MachineLearningRule) => {
+  cy.get(RUNS_EVERY_INTERVAL).clear().type(rule.runsEvery.interval);
+  cy.get(RUNS_EVERY_TIME_TYPE).select(rule.runsEvery.timeType);
+  cy.get(LOOK_BACK_INTERVAL).clear().type(rule.lookBack.interval);
+  cy.get(LOOK_BACK_TIME_TYPE).select(rule.lookBack.timeType);
+};
+
+export const expectDefineFormToRepopulateAndContinue = (rule: CustomRule) => {
```

Comment:
I think these came from a bad merge resolution; they were removed in https://github.com/elastic/kibana/pull/77090

Reply:
mmmm you are right! Thanks for catching this :)